app/service/form_task_list_service.rb

```diff
@@ -17,12 +17,16 @@ def initialize(form:, current_user:)
   end
 
   def all_sections
-    [
+    sections = [
       section_1,
-      section_2,
       section_3,
       section_4,
+      section_5,
     ]
```

Comment:
I'm wondering if maybe we consider renaming the section methods to be based on the task rather than a number 😅 Having to do all this renaming when doing a small reorder seems like a massive pain!

Reply:
Good point, David and I did discuss this and we weren't sure if it would be better to instead save it for a refactor into using the GOV.UK task list component. If we are planning on shifting over to that, then that'd be the best time to name things more appropriately

Reply:
Okay, makes sense!